examples/riot-bootloader/src/lib.rs

```diff
@@ -7,7 +7,7 @@ const RIOTBOOT_MAGIC: u32 = 0x544f_4952u32;
 type Fletcher = (u32, u32);
 
 pub fn new_fletcher() -> Fletcher {
-    (0u32, 0u32)
+    (0x0000_ffff, 0x0000_ffff)
```

Comment:
@denismerigoux can we infer the `u32` here? I thought we have to be explicit about the types if it's not `usize`.

Reply:
So apparently here the Rust parser automatically infers that this is a `u32`, probably because of the specific syntax with `0x` and the `_` to separate the bytes.